en/installconfig-images-import.md

```diff
@@ -1,41 +1,42 @@
-Title: MAAS | Select and Import Images
-TODO: Update images (pictures)
+Title: Select and Import Images | MAAS 
+TODO:  Be consistent with final version of installconfig-hwe-kernels.md (see TODO)
 
 
 # Select and Import Images
 
-This page explains how to use the web UI to select and import the images that
-MAAS requires in order to provision its nodes.
+This page explains how to select and import the images that MAAS requires in
+order to provision its nodes.
 
 Note that the MAAS CLI can also be used for image management. You must
 [Select images](manage-cli-images.md#select-images) and 
 [Import newly-selected images](manage-cli-images.md#import-newly-selected-images).
 
-By default, the most recent Ubuntu LTS release (and amd64 architecture) are
-selected. To change the selections go to the Images tab, select any other
-images (by choosing 'Release' and 'Architecture'), and click the 'Import
-images' button.
+On the Images page, you will see, by default, the three most recent Ubuntu LTS
+releases (and amd64 architecture) are selected:
 
-![import image selection](../media/import-images.png)
+![import image selection](../media/installconfig-images-import__main.png)
 
-After the import process has started, imported images will be listed beneath
-the selection area, alongside details on the image size, last update date and
-the number of nodes used by an image. A busy indicator will also appear until
-the import process has completed. 
+In the top-right corner there is a toggle switch for automatic syncing of
+images. It is highly recommended to keep this enabled.
```

Comment:
It may be helpful, though not essential, to give some idea of how frequently MAAS will check for updates, and how frequently the images change.